docs/guides/evaluate_model.ipynb

```diff
@@ -0,0 +1,210 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Evaluate the Transfer Learning\n",
+    "**This tutorial was created using alphaDIA 1.8.1 - please be aware that there might be changes in your version.**\n",
+    "\n",
+    "## 1. Prerequisites\n",
+    "The information on the training of a PeptDeep model is stored in the `stats.train.tsv` file.\n",
+    "We will be using the stats file from the dimethyl transfer learning example. You can download the file [here](https://datashare.biochem.mpg.de/s/RoyN2tUlPitJjh4) or use the file included in your alphaDIA output directory. Make sure you have python with pandas and seaborn installed.\n",
```

Comment:
Make sure you have a valid AlphaDIA environment.

Reply:
You actully don't need it for this notebook :)